### DISCUSSION TITLE
Heater Question

### DISCUSSION THREAD

**Discussion by Exit2Studios** [tagyoureit/nodejs-poolController#544] [Nixie]
I currently have my stand alone Nixie controller in my "lab" (i.e. not in production with my pool yet).

In testing, I'm not getting 24vAC when I activate the heater relay.  Everything appears to be wired correctly, so I'm not sure what would be causing this.  I want to make sure that is the expected behavior.  I have it on the same relay card as all of my valves, which all work as expected.

Also, if anyone has any pictures of the fireman's switch wired up, I would greatly appreciate it.  I have a Pentair MasterTemp 400 FWIW.

**Comment by orrious** [comment]
The fireman's switch is just a jumper you remove on the MasterTemp.  You replace it with the wires going to your relay.  You do not want to apply 24vac to the line.  It is just a switch that breaks the connection of the 24vac that the MasterTemp already creates / uses.

**Reply by Exit2Studios** [reply]
So should there be continuity between NO and C when the heater is on?

**Reply by orrious** [reply]
Correct.

**Comment by rock-crusher** [comment]
The jumper is the yellow wire

![MT 400](https://user-images.githubusercontent.com/94593954/179428098-1794ea8f-b516-416d-aadd-29c111acba46.jpg)

**Reply by Exit2Studios** [reply]
So you just remove the yellow jumper wire and replace with NO/C from the relay?  Interesting.  I don't know why I'm so scared of the heater lol.

**Reply by rstrouse** [reply]
Yes it is that simple.  Connect one wire to C and the other to NO,  You are simply putting a switch where the yellow jumper used to be.  When Nixie needs some heat to reach the setpoint it will close this relay and the heat will begin.  When the setpoint is reached Nixie will open the relay again and turn the heater off.

Set the thermostat on the heater to 104 and you will be off to the races.
